### PR TITLE
Chore/pre release 0.1.0

### PR DESCRIPTION
**Pull request by wilwade**
Various Cleanup and Chores pre-release 0.1.0

- Cleanup packages
- Add Thrift d.ts file
- Cleanup readme
- Fix class issue
- Update thrift types
- Build types

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Once you have installed the parquet.js library, you can import it as a single
 module:
 
 ``` js
-var parquet = require('parquetjs-lite');
+var parquet = require('@dsnp/parquetjs');
```
Comment (enddynayn) [inline README.md:42]: thanks!

**Review by enddynayn — APPROVED** [review]

File: lib/bloom/sbbf.ts [hunk 2]
```diff
@@ -221,7 +221,7 @@ class SplitBlockBloomFilter {
     private desiredFalsePositiveRate: number = SplitBlockBloomFilter.DEFAULT_FALSE_POSITIVE_RATE
     private numBlocks: number = 0
     private numDistinctValues: number = SplitBlockBloomFilter.DEFAULT_DISTINCT_VALUES
-    private hashStrategy = new parquet_thrift.BloomFilterHash(parquet_thrift.XxHash)
+    private hashStrategy = new parquet_thrift.BloomFilterHash(new parquet_thrift.XxHash())
```
Comment (wilwade) [inline lib/bloom/sbbf.ts:224]: @shannonwells @enddynayn This is the one line that was odd for me.

Comment (shannonwells) [reply lib/bloom/sbbf.ts:224]: That is weird. Tou shouldn't have needed to do this

File: package.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
   "name": "@dsnp/parquetjs",
   "description": "fully asynchronous, pure JavaScript implementation of the Parquet file format",
   "main": "dist/parquet.js",
+  "types": "dist/parquet.d.ts",
```
Comment (shannonwells) [inline package.json:5]: ❤️

File: lib/bloom/sbbf.ts (hunk 2 above)
Comment (wilwade) [reply lib/bloom/sbbf.ts:224]: parquet_thrift.XxHash is referring to a class. Perhaps we are using the wrong thing?

File: parquet.thrift [hunk 4]
```diff
@@ -41,9 +41,10 @@ enum Type {
 }
 
 /**
- * Common types used by frameworks(e.g. hive, pig) using parquet.  This helps map
- * between types in those frameworks to the base types in parquet.  This is only
- * metadata and not needed to read or write the data.
+ * DEPRECATED: Common types used by frameworks(e.g. hive, pig) using parquet.
```
Comment (shannonwells) [inline parquet.thrift:44]: Did you pull in an updated version of the thrift file?

**Review by shannonwells — COMMENTED** [review]
@wilwade I didn't think the hash strategy is supposed to be an instance of anything.  What led to this change?

File: parquet.thrift (hunk 4 above)
Comment (wilwade) [reply parquet.thrift:44]: I did, but it didn't change much

File: lib/bloom/sbbf.ts (hunk 2 above)
Comment (wilwade) [reply lib/bloom/sbbf.ts:224]: Looks like it is just the way that the thrift struct is translated for typescript. It all works either way.

**Pull request merged**